swap_meet/vendor.py

```diff
@@ -1,2 +1,109 @@
 class Vendor:
-    pass
\ No newline at end of file
+    '''
+    Class representing a single vendor with items to trade
+
+    Functions:
+        add(object, item)
+        remove(object, item)
+        get_by_category(object, category)
+        swap_items(object, another object, my_item, their_item)
+        swap_first_item(object, another object)
+        get_best_by_category(self, category):
+        swap_best_by_category(self, other, my_priority, their_priority):
+
+    Variables:
+        inventory: list of items for a vendor to see default is [] 
+    '''
+    def __init__(self, inventory=None):
+        if not inventory == None:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+        # self.inventory = inventory if inventory is not None else []
```

Comment:
👍  Great work in handling default parameters for mutable types. It's unsafe to set [] as the default value, but we can use None to stand in for the value when not supplied by the caller, and then use a new empty list in its place.

And yes, we can definitely use a ternary to assign the value to `self.inventory` like your comment displays.